cmake/modules/BuildJaeger.cmake

```diff
@@ -58,8 +58,6 @@ function(build_jaeger)
   endif()
   include(BuildOpenTracing)
   build_opentracing()
-  include(Buildthrift)
```

Comment:
this is not a "cleanup". i think you are addressing the regression introduced by 80e82686ebafe36fca6dfd21cb32e63ced94d5cd. could you summarize the change more accurately?